src/wp-includes/pomo/streams.php

```diff
@@ -152,6 +153,13 @@ public function close() {
 if ( ! class_exists( 'POMO_FileReader', false ) ) :
 	class POMO_FileReader extends POMO_Reader {
 
+		/**
+		 * Filename.
+		 *
+		 * @var string
```

Comment:
`$_f` is set to the return value of `fopen()`, which returns a `resource` on success, or `false` on failure.

Reply:
Good catch! Thanks. Fixed up in the original commit.